### PR TITLE
Fix calculate minimum receive after slippage

### PR DESCRIPTION
**Pull request by trungbach**
What is purpose of the changes?

 - Use bigdecimal to calculate minimum receive for universal swap

### CODE REVIEW

File: packages/universal-swap/src/handler.ts [hunk 1]
```diff
@@ -562,7 +562,7 @@ export class UniversalSwapHandler {
     try {
       const _fromAmount = toAmount(fromAmount, fromTokenOnOrai.decimals).toString();
 
-      if (!this.swapData.simulatePrice || !this.swapData.userSlippage)
+      if (!this.swapData.simulatePrice || !(this.swapData.userSlippage || this.swapData.userSlippage === 0))
```
Comment (ducphamle2) [inline packages/universal-swap/src/handler.ts:565]: We should not group condition like this. If its all || then its more readable to remove the ()

Comment (ducphamle2) [reply packages/universal-swap/src/handler.ts:565]: Also, we need an extra test case for that new condition

Comment (tucq88) [reply packages/universal-swap/src/handler.ts:565]: Also, I'd like to have this condition wrapped in a variable to provide context to the condition.

Comment (trungbach) [reply packages/universal-swap/src/handler.ts]: const isValidSlippage = this.swapData.userSlippage || this.swapData.userSlippage === 0;
if (!this.swapData.simulatePrice || !isValidSlippage)

**Review by toandq2009 — APPROVED** [review]

**Review by haunv3 — APPROVED** [review]
LGTM

**Pull request merged**